WordPress/WordPressTest/PostTests.swift

```diff
@@ -301,11 +301,11 @@ class PostTests: XCTestCase {
         XCTAssertEqual(revision.statusForDisplay(), NSLocalizedString("Local changes", comment: "Local"))
 
         revision.status = .scheduled
-        let scheduledStatusDisplay = "\(Post.title(for: .scheduled))"
+        _ = "\(Post.title(for: .scheduled))"
         XCTAssertEqual(revision.statusForDisplay(), local)
 
         revision.status = .trash
-        let trashStatusDisplay = "\(Post.title(for: .trash))"
+        _ = "\(Post.title(for: .trash))"
```

Comment:
This doesn't look right? These two variables seem unused so we should delete those lines instead?

Reply:
I will try to remove it, but I think this call could have some expected side effect that the test is expecting?

Reply:
I think it might have been an oversight: https://github.com/wordpress-mobile/WordPress-iOS/pull/11705/files#diff-2aeb6e69ee88f42fb3a015b6bf5780edL305